tests/LineBot.Tests/TestHelpers/Tools/StringTools.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
+//
+// Dirk Lemstra licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+namespace Line.Tests
+{
+    [ExcludeFromCodeCoverage]
+    public static class StringTools
+    {
+        public const string Whitespace = " ";
```

Comment:
I don't think we need this class. It is more clear to have the whitespace in the unit test instead of a variable.